sudoku-mw/src/main/webapp/WEB-INF/views/index.jsp

```diff
@@ -41,8 +41,18 @@
 
     <div class="row">
         <div id="sudoku-table" class="sudoku-table col-lg-12">
+        <div id="status_bar" class="status_bar col-md-4"></div>
         </div>
     </div>
+    &nbsp;
+    <div class="row">
+     <div class="col-md-4">
+      <p>Moves 0</p>
+     </div>
+    <div class="col-md-4">
+     <p id="timer">Time 00:00:00</p>
+    </div>
+    </div>
```

Comment:
This looks like a status bar, but then what is the reason of having  `<div id="status_bar" class="status_bar col-md-4"></div>` in line 44?